substrafl/algorithms/pytorch/torch_base_algo.py

```diff
@@ -84,6 +84,26 @@ def model(self) -> torch.nn.Module:
         """
         return self._model
 
+    @remote_data
+    def initialization(
+        self,
+        datasamples: Any,
+        shared_state: Any = None,
+    ) -> Any:
+        """This function does nothing and is useful to load the algorithm on the platform for initialization,
+        without applying any operation.
+        It is possible to overwrite this function in the algorithm and to call it within the perform round of
+        the associated strategy.
+
+        The signature of the function is constraint by the @remote_data decorator, used to load the object of
```

Comment:
Nitpick: possible typo, though I don't understand the context well enough to be sure
```suggestion
        The signature of the function is constrained by the @remote_data decorator, used to load the object of
```

Reply:
Thanks !